### PR TITLE
Update pull request template and issue templates

### PR DESCRIPTION
**Pull request by samshara**
## Addresses:
- https://github.com/IFRCGo/go-web-app/issues/1579

## Changes
- Add new PR template.
- Add issue template for bug, feature and epic request.

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

**Review by frozenhelium — APPROVED** [review]

**Review by barshathakuri — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
+## This PR Ensures:
+- [ ] No typos or grammatical errors
+- [ ] No conflict markers left in the code
+- [ ] No unwanted comments, temporary files, or auto-generated files
+- [ ] No inclusion of secret keys or sensitive data
+- [ ] No `console.log` statements meant for debugging
+- [ ] `pnpm lint` runs without errors
+- [ ] `pnpm typecheck` passes (no type errors)
+- [ ] `pnpm knip` passes
+- [ ] `pnpm build` completes successfully
```
Comment (tnagorra) [inline .github/pull_request_template.md]: Should we merge this into "CI has not failed" instead?

File: .github/ISSUE_TEMPLATE/01_bug_report.md
Comment (tnagorra) [inline .github/ISSUE_TEMPLATE/01_bug_report.md:1]: The file should be yaml

File: .github/ISSUE_TEMPLATE/02_feature_request.md
Comment (tnagorra) [inline .github/ISSUE_TEMPLATE/02_feature_request.md:1]: The file should be yaml

File: .github/ISSUE_TEMPLATE/03_epic_request.md
Comment (tnagorra) [inline .github/ISSUE_TEMPLATE/03_epic_request.md:1]: The file should be yaml

File: .github/ISSUE_TEMPLATE/01_bug_report.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+name: Report a Technical or Visual Issue on the IFRC GO Website
+description: 'Encountered a bug or glitch? Let us know so we can address it!'
+labels: ['type: bug']
```
Comment (tnagorra) [inline .github/ISSUE_TEMPLATE/01_bug_report.md]: We should also add type "Bug". Not sure if issue forms support it yet.

File: .github/ISSUE_TEMPLATE/02_feature_request.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+name: Feature Request
+description: Suggest a new idea or enhancement for the IFRC GO project.
+title: "[Feature]: "
+labels: ['type: feature-request']
```
Comment (tnagorra) [inline .github/ISSUE_TEMPLATE/02_feature_request.md]: We should also add type "Feature". Not sure if issue forms support it yet.

File: .github/ISSUE_TEMPLATE/03_epic_request.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+name: Epic
+description: Track a larger initiative with multiple related tasks and deliverables for IFRC GO project.
+title: "[Epic]: "
+labels: ['type: epic']
```
Comment (tnagorra) [inline .github/ISSUE_TEMPLATE/03_epic_request.md]: We should also add type "Feature". Not sure if issue forms support it yet.

File: .github/ISSUE_TEMPLATE/01_bug_report.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+      options:
+        - 'Low (Minor inconvenience)'
+        - 'Medium (Affects functionality, but there is a workaround)'
+        - 'High (Major functionality is broken)'
+        - 'Critical (Site is unusable)'
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
```
Comment (tnagorra) [inline .github/ISSUE_TEMPLATE/01_bug_report.md]: IMO we need a expected behavior vs actual behavior

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Review by frozenhelium — APPROVED** [review]

**Review by tnagorra — APPROVED** [review]

**Pull request merged**